config/config.txt

```diff
@@ -290,7 +290,7 @@ CHECK_RANDOMIZER
 ## from https://github.com/rg3/youtube-dl/releases
 ## or your package manager
 ## The default value assumes youtube-dl is in your system PATH
-# INVOKE_YOUTUBEDL youtube-dl
+INVOKE_YOUTUBEDL yt-dlp
```

Comment:
revert this, config should be left as is

Reply:
yea sorry gitignore was refusing to update the config.txt will do